tests/css/postcss-8-improment/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -32,7 +78,7 @@ https://github.com/postcss/postcss/releases/tag/8.0.0
 */
 
 :root {
-  --empty:  ;
+  --empty: ;
```

Comment:
Good catch :+1:

Reply:
`--empty:;` is invalid syntax
`--empty: ;` is valid syntax

So both should not be changed